README.md

```diff
@@ -0,0 +1,94 @@
+# go-billy-gluster [![GoDoc](https://godoc.org/gopkg.in/src-d/go-billy-gluster.v0?status.svg)](https://godoc.org/github.com/src-d/go-billy-gluster) [![Build Status](https://travis-ci.org/src-d/go-billy-gluster.svg)](https://travis-ci.org/src-d/go-billy-gluster) [![codecov.io](https://codecov.io/github/src-d/go-billy-gluster/coverage.svg)](https://codecov.io/github/src-d/go-billy-gluster)
+
+This package provides a basic [go-billy](https://github.com/src-d/go-billy) driver to access [gluster](https://www.gluster.org/) volumes. It uses libgfapi so it's not needed to mount the volume locally.
+
+Currently only basic actions are available, that is:
+
+* `Create`
+* `Open`
+* `OpenFile`
+* `Stat`
+* `Rename`
+* `Remove`
+
+For more information head to the [documentation](https://godoc.org/gopkg.in/src-d/go-billy-gluster.v0)
+
+# Installation
+
+This library uses [gogfapi](https://github.com/gluster/gogfapi) that need libgfapi library to run. This library is contained in the development client packages of gluster.
+
+* Arch / Manjaro:
+
+```
+pacman -S glusterfs
+```
+
+* Ubuntu / Debian:
+
+```
+apt-get install glusterfs-common
+```
+
+* CentOS / RHEL:
+
+```
+yum install glusterfs-api
+```
+
+After installing the dependency you can install the library with `go get`:
+
+```
+go get gopkg.in/src-d/go-billy-gluster.v0
+```
+
+# Example of utilization
+
+```go
+package main
+
+import (
+	"fmt"
+
+	"gopkg.in/src-d/go-billy-gluster.v0"
+)
+
+func main() {
+	fs, err := gluster.New("server", "volume")
+	if err != nil {
+		panic(fmt.Sprintf("cannot connect to gluster volume %s", err))
+	}
+
+	f, err := fs.Create("filename.ext")
+	if err != nil {
+		panic(fmt.Sprintf("cannot create file %s", err))
+	}
+
+	_, err = f.Write([]byte("text"))
+	if err != nil {
+		panic(fmt.Sprintf("cannot write to file %s", err))
+	}
+
+	err = f.Close()
+	if err != nil {
+		panic(fmt.Sprintf("cannot close file %s", err))
+	}
+
+	err = fs.Close()
+	if err != nil {
+		panic(fmt.Sprintf("cannot disconnect from volume %s", err))
+	}
+}
+```
+
+# Contribute
+
+[Contributions](https://github.com/src-d/go-billy-gluster/issues) are more than welcome, if you are interested please take a look to our [Contributing Guidelines](CONTRIBUTING.md).
+
+# Code of Conduct
+
+All activities under source{d} projects are governed by the [source{d} code of conduct](.github/CODE_OF_CONDUCT.md).
+
+# License
+
+GPL v3.0, see [LICENSE](LICENSE).
```

Comment:
GPL ?

Reply:
You're right. The newly created repo already had a GPL LICENSE and used it. Changed to Apache.

Reply:
Yep, according to the guide we use Apache for libraries.
cc @mcuadros

Reply:
Let's change it to Apache.

Reply:
It's changed in https://github.com/src-d/go-billy-gluster/pull/1/commits/e6337cd45e61d30314f3d4d9b71c1fd7221c15de